src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -80,6 +84,40 @@ protected function hook_filters() {
 			$category_translation = Tribe__Events__Integrations__WPML__Category_Translation::instance();
 			add_filter( 'tribe_events_category_slug', array( $category_translation, 'filter_tribe_events_category_slug' ), 20, 2 );
 		}
+
+		//// WORK IN PROGRESS!!!
+
+		add_action( "tribe_repository_events_pre_get_posts", function () {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: String "tribe_repository_events_pre_get_posts" does not require double quotes; use single quotes instead
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 4c880da5ee160cb0cf4c54216962696a</sub>